src/test/java/com/xellitix/jenkins/plugintool/cli/converter/PluginListOutputFormatConverterTest.java

```diff
@@ -0,0 +1,61 @@
+package com.xellitix.jenkins.plugintool.cli.converter;
+
+import com.xellitix.jenkins.plugintool.output.PluginListOutputFormat;
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * {@link PluginListOutputFormatConverter} test case.
+ *
+ * @author Garrett Ewens
+ */
+public class PluginListOutputFormatConverterTest {
+
+  // Constants
+  private static final String FORMAT_VALID = "JSON";
+  private static final String FORMAT_INVALID = "?%$^lklksdff";
+
+  // Rules
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
```

Comment:
Let's add a newline between the rules and fixtures section. It looks prettier :)